lnbits/db.py

```diff
@@ -409,28 +407,24 @@ def parse_query(cls, key: str, raw_values: list[Any], model: type[TFilterModel])
 
         if field in model.__fields__:
             compare_field = model.__fields__[field]
-            values = []
+            values: dict = {}
             for raw_value in raw_values:
                 validated, errors = compare_field.validate(raw_value, {}, loc="none")
                 if errors:
                     raise ValidationError(errors=[errors], model=model)
-                values.append(validated)
+                values[field](validated)
         else:
             raise ValueError("Unknown filter field")
 
         return cls(field=field, op=op, values=values, model=model)
 
     @property
     def statement(self):
-        if self.model and self.model.__fields__[self.field].type_ == datetime.datetime:
-            placeholder = compat_timestamp_placeholder()
-        else:
-            placeholder = "?"
         if self.op in (Operator.INCLUDE, Operator.EXCLUDE):
-            placeholders = ", ".join([placeholder] * len(self.values))
+            placeholders = ", ".join([])
```

Comment:
@jackstar12  can you maybe help me here? i cant figure out the statement in the filters with sqlalchemy 1.4 placeholders